tests/data/hashes.csv

```diff
@@ -1,6 +1,6 @@
-dummy_author/agents/dummy_aea,QmR4SuPKbe4qjgF5DVwVwHPm5eFqCx83sdavs4QEXgF1mR
-dummy_author/skills/dummy_skill,QmZFoPjg3byuvLSbpdGDzu9n7U31LMwg8idtPuk26izFPX
-fetchai/connections/dummy_connection,QmRLdNwog7rx8hrYG5DGu35BNZt2782ZMxGJXjfjJy3D4d
-fetchai/contracts/dummy_contract,QmdZvbsgYRQ2gCwAv96ZRuFhfjqz8mQkhDL3NLv9mdu5Ce
-fetchai/skills/dependencies_skill,QmZzuuX3HbpuY4niyqFp3b5jq4tmUokXknf3iyKtRo8Y7N
-fetchai/skills/exception_skill,QmWXXnoHarx7WLhuFuzdas2Pe1WCprS4sDkdaPH1w4kTo2
+dummy_author/agents/dummy_aea,QmR4SuPKbe4qjgF5DVwVwHPm5eFqCx83sdavs4QEXgF1mR
+dummy_author/skills/dummy_skill,QmZFoPjg3byuvLSbpdGDzu9n7U31LMwg8idtPuk26izFPX
```

Comment:
why have they all changed?

Reply:
the hashes haven't changed; I only set `LF` instead of `CR LF` as line ending